### ISSUE TITLE
Error Notices & Thousand Separator Issue

### ISSUE THREAD

**Issue by dademaru** [lucasstark/woocommerce-dynamic-pricing-table#1]
Hello,

I've installed the plugin and, with DEBUG enabled,
I get these **errors** which break the whole website functionality (e.g. in Checkout AJAX stops working, mini cart widget stops working, …) since they are outputted in the beginning of `<head>`.
`Notice: Undefined index: roles in /woocommerce-dynamic-pricing-table/woocommerce-dynamic-pricing-table.php on line 335`
`Notice: Undefined variable: info_message in /woocommerce-dynamic-pricing-table/woocommerce-dynamic-pricing-table.php on line 356`
`Notice: Undefined variable: info_message in /woocommerce-dynamic-pricing-table/woocommerce-dynamic-pricing-table.php on line 406`

So, while DEBUG is enabled, this plugin can't be used.

---

A second, big issue is that this plugin does not take into account the **currency settings** and **thousands separator**.
I'm using € with comma for decimals and dot for thousands (e.g. € 1.000,00).

I set pricing rules with comma (e.g. 90,00 - 80,33 - 65,60…).
Dynamic Pricing plugin has been updated time ago to take currency settings and thousands separator into account (before, you needed to use dots), so price discounts in the cart are correctly displayed.
But this plugin displays, below '**Bulk Purchase Pricing**':
€ 90,00 Per Item
€ 80,00 Per Item
€ 65,00 Per Item
rounding decimals at 0.
I've tried to revert dots in pricing rules, as in the first Dynamic Pricing releases (90.00 - 80.33 - 65.60) and the dynamic pricing table is correct, but in the cart I get € 8.033,00 - € 6.560,00…

In conclusion, the plugin should be updated to work with currency and thousands separator settings, while error notices should be fixed otherwise it can't be tested nor used.

Thanks
Davide
